host/rpi/rpi.go

```diff
@@ -45,36 +40,39 @@ var Version int
 // The actual pin mapping depends on the board revision! The default values are
 // set as the 40 pins header on Raspberry Pi 2 and Raspberry Pi 3.
 //
-// P1 is also known as J8.
+// Some header info here: http://elinux.org/RPi_Low-level_peripherals
+//
+// P1 is also known as J8 on A+, B+, 2 and later.
 var (
-	P1_1  = pin.V3_3       // max 30mA
-	P1_2  = pin.V5         // (filtered)
-	P1_3  = bcm283x.GPIO2  // High, I2C1_SDA
-	P1_4  = pin.V5         //
-	P1_5  = bcm283x.GPIO3  // High, I2C1_SCL
-	P1_6  = pin.GROUND     //
-	P1_7  = bcm283x.GPIO4  // High, GPCLK0
-	P1_8  = bcm283x.GPIO14 // Low,  UART0_TXD, UART1_TXD
-	P1_9  = pin.GROUND     //
-	P1_10 = bcm283x.GPIO15 // Low,  UART0_RXD, UART1_RXD
-	P1_11 = bcm283x.GPIO17 // Low,  UART0_RTS, SPI1_CE1, UART1_RTS
-	P1_12 = bcm283x.GPIO18 // Low,  PCM_CLK, SPI1_CE0, PWM0_OUT
-	P1_13 = bcm283x.GPIO27 // Low,
-	P1_14 = pin.GROUND     //
-	P1_15 = bcm283x.GPIO22 // Low,
-	P1_16 = bcm283x.GPIO23 // Low,
-	P1_17 = pin.V3_3       //
-	P1_18 = bcm283x.GPIO24 // Low,
-	P1_19 = bcm283x.GPIO10 // Low, SPI0_MOSI
-	P1_20 = pin.GROUND     //
-	P1_21 = bcm283x.GPIO9  // Low, SPI0_MISO
-	P1_22 = bcm283x.GPIO25 // Low,
-	P1_23 = bcm283x.GPIO11 // Low, SPI0_CLK
-	P1_24 = bcm283x.GPIO8  // High, SPI0_CE0
-	P1_25 = pin.GROUND     //
-	P1_26 = bcm283x.GPIO7  // High, SPI0_CE1
-
-	// Raspberry Pi 2 and later:
+	// Rapsberry Pi A and B, 26 pin header:
+	P1_1  pin.Pin    = pin.V3_3       // max 30mA
```

Comment:
The reason I hadn't specified the type there was to make it easier to leverage the bcm283x specific functionality of these pins but I guess it doesn't matter than much and it's quite awkward that half of the pins are pointers and the other half use interfaces.